### DISCUSSION TITLE
PyG-lib pre compiled wheels for windows

### DISCUSSION THREAD

**Discussion by rjanvier** [pyg-team/pyg-lib#278] [Q&A]
Hi!
I would like to know why there is currently no precompiled  PyG-lib for windows.
if there is anything in this topic that could need or benefits from some help from a contributor, I think you can count on me :).
Thank in advance!

**Comment by rusty1s** [comment]
I think the first step towards this goal would be to integrate Windows CI support into `pyg-lib`. We can start by adding `windows` [here](https://github.com/pyg-team/pyg-lib/blob/master/.github/workflows/python_testing.yml#L12), and go from there. Let me know if you wanna take this :)

**Reply by rjanvier** [reply]
Ok just saw your message, thank you. I will investigate in adding windows to the CI in the coming week.
